Doc/library/enum.rst

```diff
@@ -56,6 +56,7 @@ helper, :class:`auto`.
 .. class:: auto
 
     Instances are replaced with an appropriate value for Enum members.
+    Increment starts at 1 not 0.
```

Comment:
There's an extraneous line break here. It's not the increment that starts at 1, but the initial value. Also, the "not 0" text isn't needed.

Reply:
updated content